web/themes/custom/usecaselabs/templates/content/node--article--full.html.twig

```diff
@@ -0,0 +1,52 @@
+{% set layout = layout ? 'layout--' ~ layout|clean_class %}
+
+{% set classes = [
+    'node',
+    'node--type-' ~ node.bundle|clean_class,
+    node.isPromoted() ? 'node--promoted',
+    node.isSticky() ? 'node--sticky',
+    not node.isPublished() ? 'node--unpublished',
+    view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+    'page--article'
+  ]
+%}
+<article{{ attributes.addClass(classes) }}>
+	<header class="{{ layout }}">
+		<div class="header__left">
+      {{ title_prefix }}
+			<h1{{ title_attributes.addClass('node__title') }}>
+				{{ label }}
+			</h1>
+			{{ title_suffix }}
+      {% if display_submitted %}
+        <div class="node__meta">
+          {% if author_picture %}
+            <div class="node__author-image">
+              {{ author_picture }}
+            </div>
+          {% endif %}
+          <span{{ author_attributes.addClass('author-name') }}>
+            {{ 'By'|t }}
+            {% apply spaceless %}{{ author_name }}{% endapply %}
+          </span>
+          <span>{{ date }}</span>
+          {% if content.field_tags %}
+            {{ content.field_tags }}
+          {% endif %}
+        </div>
+      {% endif %}
+		</div>
+		<div class="header__image"><img src={{ file_url(content.field_image['#items'].entity.uri.value) }} alt={{ content.field_image.alt }} />
+    </div>
+	</header>
```

Comment:
Hi @imudasirali!
Suggestion: Common code can be moved to a separate twig file, and include that file here. Reference: https://github.com/axelerant-trainings/usecaselabs-drupal-frontend-development-d10/pull/5/files#diff-bd4f08bbb2de58b8bb7bbe83343c6a63adb7715a1a77db41b88487dc4012dd7a